LICENSE

```diff
@@ -1,6 +1,8 @@
 The MIT License (MIT)
 
+Copyright (c) 2013 Andrey Sitnik <andrey@sitnik.ru>
 Copyright (c) 2016 Denys
+Copyright (c) 2016 Pat Sissons <patricksissons@gmail.com>
```

Comment:
Copyright in MIT is basically pointless past calling out the initial developer. MIT is a free and clear license. Generally considered poor form to add to it, as you'll show in the contributors. It's up to the repo own wether or not to have a contributors list.